spec/generated_site_spec.rb

```diff
@@ -39,7 +39,8 @@ module JekyllPagesApi
         paths = ['baz.txt', 'quux.html', 'xyzzy.html'].sort.map do |f|
           File.join content_dir, f
         end
-        expect(Dir.glob(File.join(content_dir, '**', '*'))).to eq(paths)
+        expect(Dir.glob(File.join(content_dir, '**', '*'))).to contain_exactly(
+          *paths)
```

Comment:
Good catch! What would you think about using [`match_array`](https://www.relishapp.com/rspec/rspec-expectations/v/3-5/docs/built-in-matchers/contain-exactly-matcher) instead? Makes it _slightly_ cleaner since you don't need to have the splat (`*`).